src/main/kotlin/io/github/mojira/arisa/modules/EmptyModule.kt

```diff
@@ -0,0 +1,53 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import arrow.core.left
+import arrow.core.right
+
+data class EmptyModuleRequest(
+    val numAttachments: Int,
+    val description: String?,
+    val environment: String?
+)
+
+const val DESCDEFAULT = "Put the summary of the bug you're having here\r\n\r\n*What I expected to happen was...:*\r\nDescribe what you thought should happen here\r\n\r\n*What actually happened was...:*\r\nDescribe what happened here\r\n\r\n*Steps to Reproduce:*\r\n1. Put a step by step guide on how to trigger the bug here\r\n2. ...\r\n3. ..."
+const val ENVDEFAULT = "Put your operating system (Windows 7, Windows XP, OSX) and Java version if you know it here"
+const val MINLENGTH = 5
```

Comment:
Wouldn't those make sense as config keys?

Reply:
I was thinking about it, but at the end we dont even have this defaults anymore, so I dont think this will ever change

Reply:
Note it only goes as far as Windows 7

Reply:
Well, Mojang could update it at any point.
Is this even the same across all projects? It would be kinda weird for the BE.

For me, the default text is empty anyways, I'm not quite sure why.

Reply:
I think they removed the default text,  this is a copy from the old module but im probably going to remove it as soon as I verify that (post launch)

Reply:
Ah yes, that does make sense then